### DISCUSSION TITLE
Raspberry Pi 400 MiniDexed

### DISCUSSION THREAD

**Discussion by diyelectromusic** [probonopd/MiniDexed#592] [Show and tell]
Whilst not a particularly practical instrument (at least compared to other options), I might have found my favourite way to test MiniDexed builds now.  When using a Raspberry Pi 400 and my latest PCB I can simply use the built-in QWERTY keyboard. This means I only need power and audio out and I'm ready to go :)

Downside to this version is that it slightly obscures the uSD slot unfortunately, but I can find a workaround for that!

https://makertube.net/w/tMyv2KC5D9zFLqs32MEjQa

Kevin

![IMG_7543](https://github.com/probonopd/MiniDexed/assets/68612569/c75ba3de-deb6-423d-b5be-4d7396b8dd26)

**Comment by probonopd** [comment]
Wow. Slightly overkill I'd say (the RPi 400 would be capable of running a DAW such as Reaper and many Dexed instances plus effects) but nevertheless a cool experiment. How many MiniDexed devices and contraptions do you have in your collection nowadays? ;-)

**Reply by diyelectromusic** [reply]
Well I only have the emulated TX816 to play and one for my Lo-Fi Orchestra. The rest are just a bit of fun.

And yes, this is complete overkill.  But I still quite like it :)

Kevin
